### DISCUSSION TITLE
Chromecast with Google TV control

### DISCUSSION THREAD

**Discussion by huenemeca** [OMOTE-Community/OMOTE-Firmware#51] [General]
Could this be used to control a Chromecast with Google TV?  That would be awesome because the remote for it is so bad.

**Comment by Dark1886** [comment]
It is very possible to do that, I'm currently trying to get control of my Shield TV figured out, as some of the buttons seem to have weird behavior. See the link below to a different discussion regarding the Amazon Fire Stick. 

https://github.com/CoretechR/OMOTE/discussions/48

**Comment by ApatheticKoala** [comment]
Very new to Git so please excuse me if I am doing this incorrectly. 

This should have the information for moving this part of the project further along. https://github.com/NineWorlds/google-tv-remote

**Reply by Dark1886** [reply]
I've made some progress lately on getting the Bluetooth connectivity working with my Nvidia Shield TV Pro. Something with the way remotes work on android is a bit confusing. I believe I have everything working except the menu button, but that isn't always super important. 

If you look at my repos you can see an example of how I'm using it to control the remote. I'm hoping to work it into a more modular solution soon.
